examples/inputs/gch_multi_dim_cp_ct.yaml

```diff
@@ -0,0 +1,92 @@
+
+name: GCH multi dimensional Cp/Ct
+description: Three turbines using GCH model
+floris_version: v3.0.0
+
+logging:
+  console:
+    enable: true
+    level: WARNING
+  file:
+    enable: false
+    level: WARNING
+
+solver:
+  type: turbine_grid
+  turbine_grid_points: 3
+
+farm:
+  layout_x:
+  - 0.0
+  - 630.0
+  - 1260.0
+  layout_y:
+  - 0.0
+  - 0.0
+  - 0.0
+  turbine_type:
+  - iea_15MW_floating_multi_dim_cp_ct
+
+flow_field:
+  multidim_conditions:
```

Comment:
Is this and any other new inputs documented in the input file guide?

Reply:
Added!